aws/local.tf

```diff
@@ -11,4 +11,11 @@ locals {
        private_subnet_names = [for s in data.aws_subnet.private : s.tags["Name"]]
 
        vpc_id = data.aws_vpc.unmanaged[0].id
+
+       rolearn = "arn:aws:iam::${data.aws_caller_identity.current.account_id}:role/${var.username}"
 }
+
+
+data aws_caller_identity current { }
```

Comment:
Should this have quotations?

Reply:
Either works.   Kinda like YAML,  I like skipping the quotes when you can just infer that it is "current".    If people want to standardize on quotes we could do a cleanup PR later.

Reply:
Answering follow-on questions since GitHub won't let me reply directly:  
1. I switched rolearn to a local so we can compute it from other vars we define like account_id.  It makes the vars you do have to specify simpler.
2. Can't Approve == Same thing as Can't Reply,  GitHub frustration!

Reply:
@cslocum One last thing:  any comment on @super-cob 's question about aws/cluster-autoscaler-values.yml?   Not really sure what that was for myself or if we should keep it.

Reply:
> I switched rolearn to a local so we can compute it from other vars we define like account_id. It makes the vars you do have to specify simpler.

Makes sense, I like it.